### PR TITLE
Update query-params-str-validations.md

### PR DESCRIPTION
**Pull request by theonlykingpin**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 7cd0d57d9befc233eda3786cc52c13f1acb8ca9e at: https://6496e588df9c3f5721a94302--fastapi.netlify.app

File: docs/en/docs/tutorial/query-params-str-validations.md [hunk 1]
```diff
@@ -323,7 +323,7 @@ This specific regular expression checks that the received parameter value:
 
 If you feel lost with all these **"regular expression"** ideas, don't worry. They are a hard topic for many people. You can still do a lot of stuff without needing regular expressions yet.
 
-But whenever you need them and go and learn them, know that you can already use them directly in **FastAPI**.
+But whenever you need them, go and learn them. know that you can already use them directly in **FastAPI**.
```
Comment (lieryan) [inline docs/en/docs/tutorial/query-params-str-validations.md:326]:
```suggestion
But whenever you need them, go and learn them, know that you can already use them directly in **FastAPI**.
```

period doesn't seem correct there

**Comment by tiangolo** [comment]
📝 Docs preview for commit bf0acef75b417b4b41a0750ed414611e59f738ee at: https://6499c51a922db9009dc87c08--fastapi.netlify.app

**Comment by tiangolo** [comment]
Thanks for the interest @theonlykingpin! But actually the current phrase says what I intended, the modification alters the meaning, so I'll pass on this one. But thanks for the interest. ☕

**Pull request closed without merging**